### PR TITLE
Preparing errata bulk releases for UI

### PR DESCRIPTION
**Pull request by bklvsky**
- Adding force parameter to bulk releases errata
- Setting IN_PROGRESS status in router
- black formatting

Resolves: AlmaLinux/build-system#325

### CODE REVIEW

**Review by soksanichenko — APPROVED** [review]

File: alws/crud/errata.py [hunk 1]
```diff
@@ -1757,5 +1755,37 @@ async def reset_matched_erratas_packages_threshold(
...
-    logging.info(f'Packages for records {[record.id for record in records]}'
-                 f' have been matched if their date is later than {issued_date}')
+    logging.info(
+        f'Packages for records {[record.id for record in records]}'
+        f' have been matched if their date is later than {issued_date}'
+    )
+
+
+async def set_errata_packages_in_progress(records_ids: List[str]):
+    async with open_async_session(key=get_async_db_key()) as session:
```
Comment (anfimovdm) [inline alws/crud/errata.py:1765]: Could you please use `alws.dependencies.get_async_db_session` for session opening instead?

File: alws/routers/errata.py [hunk 2]
```diff
@@ -2,15 +2,22 @@
 
 from fastapi import APIRouter, Depends, HTTPException, Query, status
 from fastapi.responses import PlainTextResponse
-from fastapi_sqla import AsyncSessionDependency
+from fastapi_sqla import AsyncSessionDependency, open_async_session
```
Comment (anfimovdm) [inline alws/routers/errata.py:5]: Looks like unused import

**Review by anfimovdm — CHANGES_REQUESTED** [review]

File: alws/routers/errata.py (hunk 2 above)
Comment (bklvsky) [reply alws/routers/errata.py]: done

File: alws/crud/errata.py (hunk 1 above)
Comment (bklvsky) [reply alws/crud/errata.py]: done

**Review by anfimovdm — APPROVED** [review]

**Pull request merged**